include/pgduckdb/pg/declarations.hpp

```diff
@@ -30,8 +36,13 @@ typedef HeapTupleData *HeapTuple;
 
 struct Node;
 
+typedef uint16_t OffsetNumber;
+
 typedef unsigned int Oid;
 
+typedef char *Pointer;
```

Comment:
Maybe it makes sense to include `c.h`, which includes a bunch of these definitions.

Reply:
You think it's preferable rather than only bringing here what we need?

Reply:
To be clear, I think it should be completely safe to include. But it does pollute the global namespace with a bunch of things that we don't need.